source/SIL.AppBuilder.Portal.Frontend/src/ui/components/layout/index.tsx

```diff
@@ -37,6 +38,11 @@ class Layout extends React.Component<IOwnProps> {
 
     if (window.innerWidth > SM_BREAKPOINT && !isSidebarVisible) {
       showSidebar();
+    } else {
+      // intentionally do not save this state (by not using redux)
+      // this is more for dev purposes where we are constantly 
+      // resizing the browser
+      hideSidebar();
```

Comment:
why we are doing this?

Reply:
because every time I resized the window down to a smaller size, the sidebar would be in my way, when it should be hidden by default.